### ISSUE TITLE
PIN entry is not accessible using VoiceOver

### ISSUE THREAD

**Issue by n0ot** [blockchain/My-Wallet-iPhone#35]
When creating and confirming a PIN, or using a PIN to unlock a wallet, the PIN entry is not visible at all with VoiceOver. I can swipe left to right and hear that I must enter or confirm my PIN, but the number pad is neither visible by swiping around the screen, ore by tapping directly over the numbers.

I am using IOS 8
